outputs/core_pools.json

```
@@ -46,16 +45,24 @@
     "0x2d6ced12420a9af5a83765a8c48be2afcd1a8feb000000000000000000000500": "cbETH/rETH/wstETH",
     "0x3fd4954a851ead144c2ff72b1f5a38ea5976bd54000000000000000000000480": "ankrETH/wstETH-BPT",
     "0x451b0afd69ace11ec0ac339033d54d2543b088a80000000000000000000004d5": "plsRDNT-Stable",
+    "0x46472cba35e6800012aa9fcc7939ff07478c473e00020000000000000000056c": "ECLP-GHO-AUSDC",
+    "0x7272163a931dac5bbe1cb5fefaf959bb65f7346f000200000000000000000549": "ECLP-GYD-AUSDT",
+    "0x7967fa58b9501600d96bd843173b9334983ee6e600020000000000000000056e": "ECLP-wstETH-WETH",
+    "0x7b54c44fbe6db6d97fd22b8756f89c0af16202cc00000000000000000000053c": "ETHx/wstETH",
+    "0x90e6cb5249f5e1572afbf8a96d8a1ca6acffd73900000000000000000000055c": "rsETH/wETH",
     "0x9791d590788598535278552eecd4b211bfc790cb000000000000000000000498": "wstETH-WETH-BPT",
     "0xb61371ab661b1acec81c699854d2f911070c059e000000000000000000000516": "ezETH/wstETH",
     "0xc2598280bfea1fe18dfcabd21c7165c40c6859d30000000000000000000004f3": "wstETH/sfrxETH",
+    "0xcdcef9765d369954a4a936064535710f7235110a000200000000000000000558": "ECLP-weETH-wstETH",
     "0xd0ec47c54ca5e20aaae4616c25c825c7f48d40690000000000000000000004ef": "rETH/wETH BPT",
     "0xef0c116a2818a5b1a5d836a291856a321f43c2fb00020000000000000000053a": "ECLP-WOETH-WETH",
     "0x423a1323c871abc9d89eb06855bf5347048fc4a5000000000000000000000496": "4POOL",
     "0x32df62dc3aed2cd6224193052ce665dc181658410002000000000000000003bd": "RDNT-WETH",
-    "0x9f8ed1acfe0c863381b9081aff2144fc867aa7730002000000000000000004d4": "ANKR:ankrETH"
+    "0x9f8ed1acfe0c863381b9081aff2144fc867aa7730002000000000000000004d4": "ANKR:ankrETH",
+    "0x260dbd54d87a10a0fc9d08622ebc969a3bf4e6bb000200000000000000000536": "jitoSOL/wstETH"
```

```suggestion
    "0x9f8ed1acfe0c863381b9081aff2144fc867aa7730002000000000000000004d4": "ANKR:ankrETH"
```
tvl is $25k; should be removed from whitelist now -_-